### DISCUSSION TITLE
I need to implement a style plugin

### DISCUSSION THREAD

**Discussion by everfu** [hexojs/hexo#5530] [Question / Answer]
My idea is to use the tags plugin to insert a style into the style of the current page when there is a post or page that uses the current tag, can this be done? Please give me an example.

**Comment by stevenjoezhang** [comment]
Tags are rendered in post body, which is inside HTML body. But for styles (style or link element), it's better to put them in HTML head. Therefore, tag plugin is not suitable for this feature. I suggest to set a variable in post front-matter and use a filter plugin to insert the style after post rendered.
